### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ashishrawat93**
Update for RHEL template that makes the EFS mout permanent and sets environment for root user's bashrc

Use authorised under Apache License 2.0

### CODE REVIEW

File: templates/informatica-powercenter-windows.template [hunk 1]
```diff
@@ -457,6 +453,10 @@
             "us-west-2": {
                 "Partition": "aws",
                 "QuickStartS3URL": "https://s3.amazonaws.com"
+            },
```
Comment (santiagocardenas) [inline templates/informatica-powercenter-windows.template]: Please keep the ca-central-1 entry in the order that it already is (alphabetical).

**Comment by santiagocardenas** [comment]
Since we continue adding to the configuration of 16 nodes (copy/paste...copy/paste...copy/paste)... have you thought about switching over to an auto scaling group with a static size? This way the launch configuration shows up just once and it's replicated for all auto scaling group instances as they're launched.

**Comment by ashishrawat93** [comment]
Userdata script and configuration is slightly different for the first instance that gets spun up, rest all instances have the same configuration. I was under the impression that auto scaling needed all instances to have the same config.

**Comment by santiagocardenas** [comment]
You can do the first one as a seed server (outside autoscaling group) and the rest can go in autoscaling.

-Santiago

**Comment by ashishrawat93** [comment]
I will definitely explore more. We wanted to add up to 32 instances, copy/paste would have been a lot to maintain.

**Comment by abelmokadem** [comment]
Any updates on the autoscaling @ashishrawat93 ?

**Comment by ashishrawat93** [comment]
hey @abelmokadem, autoscaling was picked up by another team

**Pull request merged**